src/main/java/org/myrobotlab/arduino/Msg.java

```diff
@@ -2913,4 +2914,8 @@ public void setInvoke(boolean b){
     invoke = b;	
   }
 
+  public void setSerial(Serial serial) {

```

Comment:
why the change here?  This used to be specified in the constructor.  is there some serialization reason for this change?

Reply:
peers can be delayed in creating, to accommodate that flexibility it needs a setter